controlplane/config/crd/bases/controlplane.cluster.x-k8s.io_openshiftassistedcontrolplanes.yaml

```diff
@@ -92,6 +92,13 @@ spec:
               config:
                 description: Config specs for the OpenshiftAssistedControlPlane
                 properties:
+                  additionalEnabledCapabilities:
```

Comment:
How about using the same interface as config install? https://github.com/openshift/installer/blob/main/pkg/types/installconfig.go#L550

I think this would still meet requirement plus give us greater flexibility, as in the end this is what we are going to push through. The only thing we need to validate is to make sure we do not turn on MAPI.

Reply:
Good idea, thanks! Will update